src/NHibernate.Test/SystemTransactions/TransactionNotificationFixture.cs

```diff
@@ -200,6 +223,12 @@ public void ShouldNotifyAfterDistributedTransactionWithOwnConnection(bool doComm
 
 			Assert.That(interceptor.afterTransactionCompletionCalled, Is.EqualTo(1));
 		}
+
+		private void IgnoreIfTransactionScopeInsideSessionIsNotSupported()
+		{
+			if (!Sfi.ConnectionProvider.Driver.SupportsSystemTransactions || !TestDialect.SupportsDependentTransaction)
```

Comment:
I'm not actually sure that failed tests are related to `TestDialect.SupportsDependentTransaction`  but Postgres  gives for affected tests the following exception:
```C#
Error : NHibernate.Test.SystemTransactions.TransactionWithoutConnectionFromPrepareNotificationFixture.TwoTransactionScopesInsideOneSession
System.Transactions.TransactionAbortedException : The transaction has aborted.
  ----> System.AggregateException : One or more errors occurred. (55000: prepared transactions are disabled)
  ----> Npgsql.PostgresException : 55000: prepared transactions are disabled
Data:
  Severity: ERROR
  SqlState: 55000
  Code: 55000
  MessageText: prepared transactions are disabled
  Hint: Set max_prepared_transactions to a nonzero value.
  File: twophase.c
  Line: 336
  Routine: MarkAsPreparing
   at System.Transactions.TransactionStateAborted.EndCommit(InternalTransaction tx)
   at System.Transactions.CommittableTransaction.Commit()
   at System.Transactions.TransactionScope.InternalDispose()
   at System.Transactions.TransactionScope.Dispose()
   at NHibernate.Test.SystemTransactions.TransactionNotificationFixture.TwoTransactionScopesInsideOneSession() in /home/travis/build/nhibernate/nhibernate-core/src/NHibernate.Test/SystemTransactions/TransactionNotificationFixture.cs:line 119
--AggregateException
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at Npgsql.NpgsqlConnector.ExecuteInternalCommand(String query) in C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:line 1977
   at Npgsql.VolatileResourceManager.Prepare(PreparingEnlistment preparingEnlistment) in C:\projects\npgsql\src\Npgsql\VolatileResourceManager.cs:line 108
--PostgresException
   at Npgsql.NpgsqlConnector.<>c__DisplayClass161_0.<<ReadMessage>g__ReadMessageLong|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Npgsql.NpgsqlConnector.<>c__DisplayClass161_0.<<ReadMessage>g__ReadMessageLong|0>d.MoveNext() in C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:line 1032
--- End of stack trace from previous location where exception was thrown ---
   at Npgsql.NpgsqlConnector.ExecuteInternalCommand(FrontendMessage message, Boolean async) in C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:line 1988
```